behave_django/management/commands/behave.py

```diff
@@ -98,22 +129,118 @@ def add_behave_arguments(parser):  # noqa
             keywords['help'] = keywords['config_help']
             del keywords['config_help']
 
-        parser.add_argument(*option_strings, **keywords)
+        try:
+            parser.add_argument(*option_strings, **keywords)
+        except ArgumentError:
+            remove_option(parser, *fixed)
+            parser.add_argument(*option_strings, **keywords)
```

Comment:
Reads weird at the first try, but this is totally pythonic! :smiley: Well done. :+1:

Reply:
Thanks.